content/en/Platform Deep Dive/Collaboration/collaborate-on-pentests.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Collaborate on Pentests"
+linkTitle: "Collaborate on Pentests"
+weight: 10
+description: >
+  Work with pentesters and your team.
+---
+
+{{% pageinfo %}}
+<span style="color:#8348D3">**BETA FEATURE**</span><br>
```

Comment:
@1roshan We can potentially add something more about this feature being in beta.